cypress/e2e/orangeHRM/candidateTestCases.ts

```diff
@@ -0,0 +1,52 @@
+import { apiHelper } from "../../support/Helpers/apiHelper";
+import DashboardPage from "../../support/Page Object Models/DashboardPage";
+import RecruitmentPage from "../../support/Page Object Models/RecruitmentPage";
+import ScheduleInterviewPage from "../../support/Page Object Models/ScheduleInterviewPage";
+import ViewCandidatePage from "../../support/Page Object Models/ViewCandidatePage";
+import LoginHomePage from "../../support/Page Object Models/loginHomePage";
+import { InterviewStatus } from "../../support/Helpers/uiHelper";
+Cypress.on('uncaught:exception', (err, runnable) => {
+    return false
+})
+
+describe("Tests for Candidates Tab", () => {
+    const loginPageObj: LoginHomePage = new LoginHomePage();
+    const dashboardPageObj: DashboardPage = new DashboardPage();
+    const recruitmentPageObj: RecruitmentPage = new RecruitmentPage();
+    const viewCandidatePageObj: ViewCandidatePage = new ViewCandidatePage();
+    const scheduleInterviewPageObj: ScheduleInterviewPage = new ScheduleInterviewPage();
+
+    beforeEach(() => {
+        cy.fixture("employeeInfo").as("empInfo")
+        cy.fixture("interviewDetails").as("interviewDetails")
+        cy.visit('web/index.php/auth/login')
```

Comment:
we always open this for the login step, right? so you can move it to the command